eel/__init__.py

```diff
@@ -26,6 +26,9 @@
 
 
 mimetypes.add_type('application/javascript', '.js')
+# Allows to apply the CSS files, without this line the
+# CSS files are loaded but not applied on the page.
+mimetypes.add_type('text/css', '.css')
```

Comment:
I'm wondering whether manually adding this to mimetypes' map is really needed... With Python 3.9 (which you mentioned in issue #684) when I run
```python
import mimetypes
print(mimetypes.guess_type("foo.css")[0])
```
I get the correct response, "text/css" (in fact, I also get that for "*.js", which makes me wonder whether that could be removed...
I also fail to reproduce that part on my machine, both on Windows and Linux just adding '.css' to allowed_extensions is sufficient to get the CSS files to load correctly.

Reply:
Hi, sorry for the very late reply. I myself had no problem with EEL, but it seems that on certain configurations, CSS files are not detected as `text/css`. I don't know if this was a version-specific bug or not.